### DISCUSSION TITLE
Poor performance with FF on Synology DSM 11 Docker + MariaDB

### DISCUSSION THREAD

**Discussion by lukou31** [firefly-iii/firefly-iii#7893] [Firefly III]
Hello,

I have poor performance when I add a new expense en FF. I use FF on a Synology NAS DS920+, docker (latest version) and Synology buildin MariaDB database.  About 3s - 5s to save a new item !

Do I have to change parameters on the DB (index, ...) to increase performance ? 
Cu

**Comment by JC5** [comment]
I don't know, what do the logs show?

**Comment by lukou31** [comment]
Since the latest deployment, performances are so cool. My wife thanks to you and it is now very efficient to make budget.

Thanks !

**Reply by JC5** [reply]
That's great to hear!
